src/test/suite/completion.test.ts

```diff
@@ -438,6 +438,26 @@ describe("Should do completion", () => {
       stdinMock.write(new SuggestionShownRequestMatcher("dom"), "utf8")
     ).once();
   });
+  it("should report suggestion shown only once for the same suggestion", async () => {
```

Comment:
im starting to get the feeling that we should split this file, lets move all debounce related tests to a separate test file?

Reply:
will be handled in a different pr - https://tabnine.atlassian.net/browse/DEV2-2078